include/faabric/transport/MessageEndpointServer.h

```diff
@@ -67,6 +67,8 @@ class MessageEndpointServer
   private:
     const int port;
 
+    std::unique_ptr<RecvMessageEndpoint> endpoint = nullptr;
```

Comment:
This fell out of the refactor away from using the message context to shut down. Now the endpoint server can manage its own endpoint internally.